securitylevel.go

```diff
@@ -0,0 +1,38 @@
+// Copyright 2020 Contributors to the Veraison project.
+// SPDX-License-Identifier: Apache-2.0
+
+package eat
+
+import "fmt"
+
+/*
+security-level-type = &(
+    unrestricted: 1,
+    restricted: 2,
+    secure-restricted: 3,
+    hardware: 4
+)
+*/
+const (
```

Comment:
just a suggestion, only if you really feel like it: to document the [semantics](https://ietf-rats-wg.github.io/eat/draft-ietf-rats-eat.html#section-3.7) on each constant.  this also allows us to double check what is in the spec and possibly provide feedback to Laurence on missing / unclear bits, etc.